kafka-connect-mqtt/src/main/scala/com/datamountaineer/streamreactor/connect/mqtt/sink/MqttWriter.scala

```diff
@@ -60,18 +71,22 @@ class MqttWriter(client: MqttClient, settings: MqttSinkSettings)
         kcqls.map(k => {
           //for all the records in the group transform
           records.map(r => {
```

Comment:
@Morphiaaa, @tfreiling-livongo Please add some documentation here on what specifically we customized here.

I think it's a better idea in the long run to clone `MqttWriter.scala` to a separate file, e.g. `LvMqttWriter.scala` and make our changes there.  And then use that to build our own `kafka-connect-livongo-mqtt` JAR file.  If that's too complex to do right now, then let's file a JIRA ticket to work on that later.

Reply:
I added LivongoMqttWriter.scala as you mentioned.  I also added comments.